src/js/components/PageHeader.js

```diff
@@ -121,7 +131,8 @@ PageHeader.propTypes = {
   iconClassName: React.PropTypes.oneOfType([
     React.PropTypes.string,
     React.PropTypes.object,
-  ])
+  ]),
+  subTitleClassName: React.PropTypes.string,
```

Comment:
Can we allow `subTitleClassName` to be a string or object in order to harness the full ability of `classnames`? (see above proptype declarations)